azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -122,6 +122,7 @@
    * supported, use comma to separate the values, values are case insensitive.
    */
   private static final String EXTRA_HCAT_LOCATION = "other_hcat_location";
+  private static final String EXTRA_HCAT_CLUSTERS = "other_hcat_clusters";
```

Comment:
Update documentation to provide context around this (so the reader knows how it differes from EXTRA_HCAT_LOCATIONS). Also, we should mark EXTRA_HCAT_LOCATIONS as deprecated and ask users to switch to *_CLUSTERS to take advantage of Metastore HA.